src/Symfony/Component/Cache/Adapter/CouchbaseBucketAdapter.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Symfony\Component\Cache\Exception\CacheException;
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+use Symfony\Component\Cache\Marshaller\DefaultMarshaller;
+use Symfony\Component\Cache\Marshaller\MarshallerInterface;
+
+/**
+ * @author Antonio Jose Cerezo Aranda <aj.cerezo@gmail.com>
+ */
+class CouchbaseBucketAdapter extends AbstractAdapter
+{
+    private const THIRTY_DAYS_IN_SECONDS = 2592000;
+    private const MAX_KEY_LENGTH = 250;
+    private const KEY_NOT_FOUND = 13;
+    private const VALID_OPTIONS_DSN = [
+        'operationTimeout',
+        'configTimeout',
+        'configNodeTimeout',
+        'n1qlTimeout',
+        'httpTimeout',
+        'configDelay',
+        'htconfigIdleTimeout',
+        'durabilityInterval',
+        'durabilityTimeout',
+    ];
+
+    /** @var \CouchbaseBucket */
+    private $bucket;
+
+    /** @var MarshallerInterface */
+    private $marshaller;
+
+    /**
+     * CouchbaseBucketAdapter constructor.
+     *
+     * @throws CacheException
+     */
+    public function __construct(
```

Comment:
Please use long lines for the method signature

Reply:
Hi @OskarStark,

This is done.